src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -82,11 +97,14 @@ public Tenant save(UUID consortiumId, Tenant tenantDto) {
 
     checkTenantNotExistsAndConsortiumExistsOrThrow(consortiumId, tenantDto.getId());
     Tenant savedTenant = saveTenant(consortiumId, tenantDto);
+    User shadowAdminUser = userTenantService.prepareShadowUser(adminUserId, currentTenantContext.getTenantId());
 
-    runInFolioContext(createFolioExecutionContextForTenant(tenantDto.getId(), currentTenantContext, folioMetadata),
-      () -> configurationClient.saveConfiguration(createConsortiaConfigurationBody(centralTenantId)));
+    try (var context = new FolioExecutionContextSetter(prepareContextForTenant(tenantDto.getId(), folioModuleMetadata, folioExecutionContext))) {
+      userTenantRepository.save(createUserTenantEntity(consortiumId, shadowAdminUser, tenantDto));
```

Comment:
we shoud save user_tenant entity in scope of central tenant, not institutional tenant

Reply:
fixed